server/src/routes/project.ts

```diff
@@ -15,6 +15,13 @@ projectRoutes.route("/").get(
         req.query[key] = parseInt(req.query[key]);
       }
     });
+    const { categories } = req.query;
```

Comment:
Shouldn't this query parameter by singular? so category instead of categories?

Reply:
So I was testing this and the way the project route is written if you use the singular since we're passing in the req.query to the prisma statement it won't overwrite the category field and it'll throw an error

Reply:
We can either refactor Nick's code or I can just like delete the category key from the query after I add the req.query.categories attribute

Reply:
Yeah, so I think it's best if you create a separate filter object, and only create the fields on that object when the query field is not null. Since categories query will only be _one_ category. Yeah you can go ahead and refactor Nick's code